src/main/java/org/apache/commons/compress/archivers/ar/ArArchiveInputStream.java

```diff
@@ -147,24 +140,22 @@ public ArArchiveInputStream(final InputStream inputStream) {
         super(inputStream, StandardCharsets.US_ASCII.name());
     }
 
-    private int asInt(final byte[] byteArray, final int offset, final int len) throws IOException {
-        return asInt(byteArray, offset, len, 10, false);
+    private int asOptionalDecimal(final byte[] byteArray, final int offset, final int len) throws IOException {
```

Comment:
I don't understand these method name changes. What's optional if there's always a return value? Either the method names can remain the same, or some Javadoc would be helpful. WDYT?

Reply:
Fixed in https://github.com/apache/commons-compress/pull/685/commits/27f247a40d2f984b2dab79a7860b1d2e69088421